store/views.py

```diff
@@ -12,58 +15,71 @@ def index(request):
 
 def bookDetailView(request, bid):
     template_name = 'store/book_detail.html'
+    book1=Book.objects.get(pk=bid)
```

Comment:
This may fail with invalid book ID given in POST request, and would lead to server error. Expected behavior is to inform user with Not found (404) error. You have imported `get_object_or_404` but haven't used that.